### PR TITLE
Remove Configure Filter Manager from CAVE Tools menu because it doesn…

### PR DESCRIPTION
**Pull request by tiffanycmeyer13**
…'t work

### CODE REVIEW

**Comment by srcarter3** [comment]
Hmm... so one of the menu items in the "Plots" submenu works still?

If that's the case, it seems like it should just be an item in the Tools menu directly, instead of having a "Plots" submenu with only one option?

**Comment by tiffanycmeyer13** [comment]
Yeah...I was kinda thinking that too

File: viz/gov.noaa.nws.ncep.viz.overlays/plugin.xml [hunk 1]
```diff
@@ -66,24 +66,6 @@
             </command>
             <menu
                    label="Plots">
```
Comment (srcarter3) [inline viz/gov.noaa.nws.ncep.viz.overlays/plugin.xml]: So, you should be able to remove this `<menu` section... and then just add a single menu item, like the pgen one right above, using a `<menuContribution` section.  And try the `locationURI="menu:tools?after=P"` and see if that puts it in the right place... if not, I can help you get it in the right place

**Comment by srcarter3** [comment]
Okay sweet, so that is still in the proper alphabetical order, ya?

**Comment by tiffanycmeyer13** [comment]
Yep 
image

**Comment by srcarter3** [comment]
Awesome!

**Pull request merged**